### PR TITLE
Update home, software, dataset pages

### PR DESCRIPTION
**Pull request by desh2608**
- Use Google Apps Script to get data from Google Sheets for Software and Dataset pages
- Change website theme from Minimal to Dinky for wider column (better for tables)
- Updated sections on index page: add Publications, Events in reverse chronological order

### CODE REVIEW

File: datasets.md [hunk 1]
```diff
@@ -1,72 +1,31 @@
...
-| Sheffield Wargames Corpus                         | cocktail party                       | UK English                     | spontaneous                        | 7                  | 92                  | 3          | 2013        | free                         | to add |
-| DIRHA                                             | domestic                             | various                        | command, read, spontaneous         | 11                 | 40                  | no         | 2014        | free (partial avail.)        | to add |
-| CHiME 3                                           | public spaces                        | US English                     | read                               | 48                 | 6                   | no         | 2015        | free given WSJ0 (1.5 k$)     | to add |      |
+## Submitting a new dataset
 
+To submit a new dataset, please use the following form:
 
+<iframe src="https://script.google.com/macros/s/AKfycbzQcBYOyjDJh3U3HijedXClUfLgajavmlNDOQXgvjhiqWqdR5MRuEg7a8UtVjOEqu86/exec"
```
Comment (mdelcroix) [inline datasets.md:17]: Great to have the form to add new entries.
Is it possible to give pre-defined categories for Use-case, speaking style etc
We could also add others and then the user needs to input manually. 
I don't know if it is easy or possible to do.
Use-case: Noise reduction, Speech separation, Noise-robust ASR, Meeting ASR, Speaker recognition, Diarization 
speaking style: Command, Read speech, Spontaneous, conversation

The above is just an idea of potential categories, but there may be more or more adequate ones.

File: index.md [hunk 2]
```diff
@@ -14,23 +14,25 @@ See the SIG constitution for more information. Membership is free of charge and
...
 
-# [Datasets](https://isca-sig-rosp.github.io/ISCA-SIG-RoSP/datasets.html)
+# [Datasets](/datasets.md)
 
 This page provides a list of datasets, with corresponding software baselines and evaluation results.
 
 # [Software](https://wiki.inria.fr/rosp/Software)
 This page provides a list of speech processing software and corresponding application recipes for different datasets.
 
 # SIG events
```
Comment (mdelcroix) [inline index.md:26]: Should we also create a google sheet for events?
That may be easier to maintain?

**Review by mdelcroix — APPROVED** [review]
Great progress!
Thanks a lot. I have just minor comments that we can discuss later if needed.

File: datasets.md (hunk 1 above)
Comment (desh2608) [reply datasets.md:17]: Yes, this should be doable. I'll add it.

Comment (desh2608) [reply datasets.md]: I updated the form to use Google Forms instead, which has a much nicer interface. You can see the form here: https://forms.gle/PBQ5ipjqTEsDwhYZ9

File: index.md (hunk 2 above)
Comment (desh2608) [reply index.md:26]: I'll add a TODO for this for the future. In the meantime, I suppose we can merge these changes for now.

**Pull request merged**